monkestation/code/modules/cybernetics/augments/arm_augments/item_sets/jobs.dm

```diff
@@ -0,0 +1,179 @@
+		/obj/item/knife/kitchen,
+		/obj/item/reagent_containers/cup/beaker
+	)
+	encode_info = AUGMENT_NT_LOWLEVEL
+
+/obj/item/organ/internal/cyberimp/arm/item_set/janitor
+	name = "janitorial toolset implant"
+	desc = "A set of janitorial tools hidden behind a concealed panel on the user's arm."
+	items_to_create = list(
+		/obj/item/lightreplacer,
+		/obj/item/holosign_creator,
+		/obj/item/soap/nanotrasen,
+		/obj/item/reagent_containers/spray/cyborg_drying,
+		/obj/item/mop/advanced,
+		/obj/item/paint/paint_remover,
+		/obj/item/reagent_containers/cup/beaker/large,
+		/obj/item/reagent_containers/spray/cleaner
+	)
+	encode_info = AUGMENT_NT_LOWLEVEL
+
+/obj/item/organ/internal/cyberimp/arm/item_set/janitor/emag_act()
+	if(obj_flags & EMAGGED)
+		return FALSE
+	for(var/datum/weakref/created_item in items_list)
+	to_chat(usr, span_notice("You unlock [src]'s integrated deluxe cleaning supplies!"))
+	items_list += WEAKREF(new /obj/item/soap/syndie(src)) //We add not replace.
+	items_list += WEAKREF(new /obj/item/reagent_containers/spray/cyborg_lube(src))
+	obj_flags |= EMAGGED
+	return TRUE
+
+/obj/item/organ/internal/cyberimp/arm/item_set/detective
+	name = "detective's toolset implant"
+	desc = "A set of detective tools hidden behind a concealed panel on the user's arm."
+	items_to_create = list(
+		/obj/item/evidencebag,
+		/obj/item/evidencebag,
+		/obj/item/evidencebag,
+		/obj/item/detective_scanner,
+		/obj/item/lighter
+	)
+	encode_info = AUGMENT_NT_HIGHLEVEL
+
+/obj/item/organ/internal/cyberimp/arm/item_set/detective/Destroy()
+	on_destruction()
+	return ..()
+
+/obj/item/organ/internal/cyberimp/arm/item_set/detective/proc/on_destruction()
+	//We need to drop whatever is in the evidence bags
+	for(var/obj/item/evidencebag/baggie in contents)
+		var/obj/item/located = locate() in baggie
+		if(located)
+			located.forceMove(drop_location())
+
+/obj/item/organ/internal/cyberimp/arm/item_set/chemical
+	name = "chemical toolset implant"
+	desc = "A set of chemical tools hidden behind a concealed panel on the user's arm."
+	items_to_create = list(
+		/obj/item/reagent_containers/cup/beaker,
+		/obj/item/reagent_containers/cup/beaker,
+		/obj/item/reagent_containers/cup/beaker,
+		/obj/item/reagent_containers/dropper
+	)
+	encode_info = AUGMENT_NT_HIGHLEVEL
+
+/obj/item/organ/internal/cyberimp/arm/item_set/atmospherics
+	name = "atmospherics toolset implant"
+	desc = "A set of atmospheric tools hidden behind a concealed panel on the user's arm."
+	items_to_create = list(
+		/obj/item/extinguisher,
+		/obj/item/analyzer,
+		/obj/item/crowbar,
+		/obj/item/holosign_creator/atmos
+	)
+	encode_info = AUGMENT_NT_HIGHLEVEL
+
+/obj/item/organ/internal/cyberimp/arm/item_set/combat
+	name = "combat cybernetics implant"
+	desc = "A powerful cybernetic implant that contains combat modules built into the user's arm."
+	items_to_create = list(
+		/obj/item/melee/energy/blade/hardlight,
+		/obj/item/gun/medbeam,
+		/obj/item/borg/stun,
```

Comment:
this implant contains a 3.3 crit hardlight blade AND a **instant hardstun** borg arm that has infinite charge due to no cell i think AND its crew researchable, so its not antag restricted, needless to say this will probably get abused by powergamers or something since 1v1s have no counterplay other than keeping your distance or getting one too and striking first. I didnt actually get into a fight with this but in 1v1 it will probably fuck someone up.

Reply:
no longer instant sun and prob gonna remove the hardlight

Reply:
Could make a weaker version of the hardlight maybe, I'm unsure about the medibeam though since normally the only way crew get those is through a mech.